api/src/main/java/com/codeforcommunity/rest/subrouter/TeamsRouter.java

```diff
@@ -70,7 +72,7 @@ private void registerGetTeam(Router router) {
   private void handleGetTeamRoute(RoutingContext ctx) {
     JWTData userData = ctx.get("jwt_data");
     int teamId = RestFunctions.getRequestParameterAsInt(ctx.request(), "team_id");
```

Comment:
When getting a Team by Id it should also include the Goals for that team. New naming makes that clear.